torchrl/modules/distributions/discrete.py

```diff
@@ -469,3 +467,57 @@ def rsample(self, sample_shape: Union[torch.Size, Sequence] = None) -> torch.Ten
             raise ValueError(
                 f"Unknown reparametrization strategy {self.reparam_strategy}."
             )
+
+
+class Ordinal(D.Categorical):
```

Comment:
Note: we _could_ have a masked version of this distribution (although it is rare to have masks/illegal actions for continuously supported distributions -- hence since this is used after a discretisation layer, the usefulness of supporting masks is imo arguable). Anyway, would leave it FAR.